master/pkg/model/defaults.go

```diff
@@ -23,13 +23,27 @@ const (
 	NoneCheckpointPolicy = "none"
 )
 
+// Default task environment docker image names.
+const (
+	defaultCPUImage = "determinedai/environments:py-3.6.9-pytorch-1.4-tf-1.15-cpu-aaa3750"
+	defaultGPUImage = "determinedai/environments:cuda-10.0-pytorch-1.4-tf-1.15-gpu-aaa3750"
+)
+
 // DefaultExperimentConfig returns a new default experiment config.
-func DefaultExperimentConfig() ExperimentConfig {
+func DefaultExperimentConfig(taskContainerDefaults *TaskContainerDefaultsConfig) ExperimentConfig {
 	defaultDescription := fmt.Sprintf(
 		"Experiment (%s)",
 		petname.Generate(TaskNameGeneratorWords, TaskNameGeneratorSep))
 
-	return ExperimentConfig{
+	cpuImage := defaultCPUImage
+	gpuImage := defaultGPUImage
+
+	if taskContainerDefaults != nil && taskContainerDefaults.Image != nil {
+		cpuImage = taskContainerDefaults.Image.CPU
+		gpuImage = taskContainerDefaults.Image.GPU
+	}
```

Comment:
non-blocking: why not just put this at the bottom with the other `if taskContainerDefaults != nil` clause?  Or move that clause up here and take a similar strategy?